admin/views/check-form/view-assign-forms-to-check-windows.ejs

```diff
@@ -0,0 +1,44 @@
+<div class="grid-row">
+  <div class="column-two-thirds">
+    <%- partial('partials/_breadcrumbs', { breadcrumbs: breadcrumbs }) %>
+  </div>
+  <div class="column-one-third">
+    <%- partial('partials/_name_and_sign_out') %>
+  </div>
+</div>
+
+<div class="grid-row">
+  <div class="column-two-thirds">
+    <h1 class="heading-xlarge"><%= pageTitle %></h1>
+    <p class="lede" id="lead-paragraph">
+      Select forms to assign to each window. Each form can be used for multiple windows.
+    </p>
+    <% if (Array.isArray(checkWindowData) && checkWindowData.length > 0) { %>
+    <span class="bold-small">
+            <p>Check Window</p>
+    </span>
+    <div class="assigned-check-windows">
+      <% checkWindowData.forEach(cw => { %>
+      <dl>
+        <dd>
+          <p class="font-small"><%= cw.name %></p>
+          <div><span class="bold-small"><a href="/check-form/select-form/familiarisation/<%= cw.urlSlug %>">'Try it out' period</a></span>
+            (<%= cw['FamiliarisationCheckFormCount'] === 1 ? `${cw['FamiliarisationCheckFormCount']} form` : `${cw['FamiliarisationCheckFormCount']} forms` %> assigned)</div>
+          <p><%= cw.familiarisationCheckStartDate.format('D MMMM') %> to <%= cw.familiarisationCheckEndDate.format('D MMMM YYYY') %></p>
```

Comment:
I think the formatted date should be passed in from the Presenter.  Also shoudn't the date formatting should be a method in the date service?